locales/en-US/app.l20n

```diff
@@ -95,3 +95,7 @@
 <tourDoneButton "Done">
 <userCountContainer "There are <span class=\"bold\">{{installation_count}}</span> people trying {{title}} right now!">
 <highlightPrivacy "Your privacy">
+
+<noScriptHeading "Uh oh...">
```

Comment:
@lmorchard you want me to back that stuff out or leave it for now and file a follow up bug?